### PR TITLE
Add warning dialog for too high iPad count

### LINKED ISSUE #114: Add warning if iPad count > 10

### PR DESCRIPTION
**Pull request by GodVenn**
Closes #114

### CODE REVIEW

File: frontend/src/Order.tsx [hunk 1]
```diff
@@ -99,8 +103,15 @@ const Order = () => {
         console.log(response)
     }
 
+    const onClickCreate = async () => {
```
Comment (VegardOztan) [inline frontend/src/Order.tsx:106]: Change name to `onClickSubmit` ?

File: frontend/src/components/CustomDialogs.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                    <Icon data={warning_filled} size={32} color={tokens.colors.interactive.warning__resting.rgba}></Icon>
+                    <Typography color="warning" style={{ marginLeft: '6px', marginTop: '7px' }}>
+                        Warning!
+                    </Typography>
+                </Wrapper2>
+            </Dialog.Title>
+            <Dialog.CustomContent>
+                <Typography variant="body_short">
+                    <span>
+                        {' '}
```
Comment (VegardOztan) [inline frontend/src/components/CustomDialogs.tsx:64]: Can be removed

File: frontend/src/Order.tsx (hunk 1 above)
Comment (GodVenn) [reply frontend/src/Order.tsx]: Fixed

File: frontend/src/components/CustomDialogs.tsx (hunk 2 above)
Comment (GodVenn) [reply frontend/src/components/CustomDialogs.tsx]: Fixed

**Review by VegardOztan — APPROVED** [review]

**Review by VegardOztan — APPROVED** [review]

**Pull request merged**